### ISSUE TITLE
unable to use the code

### ISSUE THREAD

**Issue by wherejm39** [AlphaSlayer1964/kemono-dl#143]
First, I'm totally noob about Python and any other c language.
I download python 3.10.7 and also the code from here.
When I install Python, it doesn't show the option which disable path length limit so I manually disable it by editing regisrty.
After I try to open python by clicking kemono-dl.py file, it automatically shut down pyrhon window so I use powershell to download requirements.
After that I type python kemono-dl.py --cookies="cookies.txt" -- links ~~~~~~~ on powershell and at the end, it showed up like this
![화면 캡처 2022-10-03 223816](https://user-images.githubusercontent.com/50871012/193591654-ac670d55-a362-433e-a74c-698882d8f401.png)
I'm korean so last message is 'This connection is blocked by online host.
I have no idea about what to do and I saw simular issue and try by that way but still don't work.

**Comment by ovear** [comment]
This usually means your ISP just block kimono.

**Comment by wherejm39** [comment]
Is it possible to get through it by vpn?

**Comment by rktfier** [comment]
Of course, you can bypass restrictions even by changing your DNS if you're lucky.

**Comment by wherejm39** [comment]
ok I'll try with VPN Thx!
